### PR TITLE
Coroutines + standard ethree patch

### PR DESCRIPTION
**Pull request by BavovnaIsTaken**

### CODE REVIEW

File: ethree/build.gradle [hunk 1]
```diff
@@ -9,7 +9,7 @@ apply plugin: 'maven'
 apply plugin: 'org.jetbrains.dokka'
 
 group 'com.virgilsecurity'
-version '0.3.0'
+version '0.3.01'
```
Comment (andrii-iakovenko) [inline ethree/build.gradle]: 0.3.1

File: ethree/src/main/java/com/virgilsecurity/android/ethree/kotlin/interaction/EThree.kt [hunk 2]
```diff
@@ -192,7 +194,8 @@ class EThree
                             }
                         }
             } catch (throwable: Throwable) {
-                if (throwable is KeychainEntryAlreadyExistsWhileStoringException)
+                if (throwable is EntryAlreadyExistsException
```
Comment (andrii-iakovenko) [inline ethree/src/main/java/com/virgilsecurity/android/ethree/kotlin/interaction/EThree.kt]: Wrong code formatting

**Review by andrii-iakovenko — CHANGES_REQUESTED** [review]

File: ethree/build.gradle (hunk 1 above)
Comment (BavovnaIsTaken) [reply ethree/build.gradle]: But I was thinking about that .x gives less possible patches. What if we have to make more than 10 patches?

File: ethree/src/main/java/com/virgilsecurity/android/ethree/kotlin/interaction/EThree.kt (hunk 2 above)
Comment (BavovnaIsTaken) [reply ethree/src/main/java/com/virgilsecurity/android/ethree/kotlin/interaction/EThree.kt]: agree

**Review by andrii-iakovenko — APPROVED** [review]

**Pull request merged**